context.cpp

```diff
@@ -321,6 +306,7 @@ namespace Sass {
 
   string Context::format_source_mapping_url(const string& file)
   {
+/*
```

Comment:
This is just temporary. Will have to do the compressed mode first and than add back source-maps. At least I got some unit test for this part!